client/src/components/employer-dashboard-component/employer-dashboard-table/index.jsx

```diff
@@ -0,0 +1,83 @@
+import PropTypes from "prop-types";
+import { Chip, Avatar } from "@mui/material";
+import { DataGrid } from "@mui/x-data-grid";
+
+function EmployerDashboardTable({
+  slicedData,
+  page,
+  rowsPerPage,
+  handleChangePage,
+  handleChangeRowsPerPage,
+  count,
+}) {
+  const formatDate = (date) => {
+    return `${date.getMonth() + 1}/${date.getDate()}/${date.getFullYear()}`;
+  };
+
+  const columns = [
+    { field: "employerName", headerName: "Employer Name", width: 200 },
+    {
+      field: "date",
+      headerName: "Date",
+      width: 200,
+      valueFormatter: ({ value }) => formatDate(value),
+    },
+    { field: "phoneNumber", headerName: "Phone Number", width: 200 },
+    { field: "email", headerName: "Email", width: 200 },
+    { field: "primaryContact", headerName: "Primary Contact", width: 200 },
+    {
+      field: "owner",
+      headerName: "Owner",
+      width: 150,
+      renderCell: (params) => (
+        <Chip
+          avatar={<Avatar>{/* logic for the avatar */}</Avatar>}
+          label={params.value}
+        />
+      ),
+    },
+  ];
+
+  const rows = slicedData.map((item, index) => ({
+    id: index,
+    employerName: item.employerName,
+    date: item.date,
+    phoneNumber: item.phoneNumber,
+    email: item.email,
+    primaryContact: item.primaryContact,
+    owner: item.owner,
+  }));
+
+  return (
+    <div style={{ minHeight: 500, minWidth: "80%" }}>
```

Comment:
We can remove this div and it will enable the grid to fill the remaining part of the row all the way to the right margin. On wider screens there becomes a large gap
![Screenshot 2024-01-21 at 13 13 21](https://github.com/uoftblueprint/tces/assets/41130718/ab9e04ee-b0c0-4acf-b5a3-5c7492b8675a)
You can take a look at Kevin's page, `client/src/components/job-lead-dashboard-component/job-lead-dashboard-table/index.jsx` He has the same implementation as you do here